docs/examples/evaluation/prometheus_evaluation.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>NIT: can you add in brackets beside 77% the calculation? Actually, shouldn't we have one for faithfulness and another for relevancy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
Yeah sure. It's almost close so added higher one. Will add them seperately.